### PR TITLE
Blog on magecart hacks

### PR DESCRIPTION
**Pull request by sammacbeth**
Included the list of affected sites in a csv file - not sure the best format for this. I think the list is probably too long to go in the article itself.

### CODE REVIEW

File: blog/trackers-who-steal.md [hunk 1]
```diff
@@ -0,0 +1,216 @@
+title: The Trackers who steal
```
Comment (remusao) [inline blog/trackers-who-steal.md]: Should this be title case?

File: blog/trackers-who-steal.md [hunk 2]
```diff
@@ -0,0 +1,216 @@
...
+header_img: blog/blog-cc-stealing.png
++++
+
+We're all aware of the trackers siphoning off information about you as you browse the web. These trackers are mostly doing this for some business intelligence related reason - websites use these services to try to 'better understand' their customers, or to target them in order to attract their attention in a way which will benefit that website owner - be-it increasing the value of products customers put into their shopping cart, or increasing the likelihood that they click an ad.
+
+However, there is another kind of tracker which is more nefarious than these. These are hidden scripts placed by hackers on E-commerce sites which try to steal your credit-card details as you enter them. In the last year a string of attacks -- dubbed 'Magecart' -- have affected major sites, including [British Airways](https://www.riskiq.com/blog/labs/magecart-british-airways-breach/), [Ticketmaster](https://www.riskiq.com/blog/labs/magecart-ticketmaster-breach/), [NewEgg](https://www.riskiq.com/blog/labs/magecart-newegg/) and [VisionDirect](https://twitter.com/troyhunt/status/1064069833967337472); stealing payment information from millions of consumers.
```
Comment (remusao) [inline blog/trackers-who-steal.md]: > In the last year a string of attacks

Should there be a coma after "In the last year"?

File: blog/trackers-who-steal.md [hunk 3]
```diff
@@ -0,0 +1,216 @@
...
+
+At WhoTracks.Me we are monitoring the third-parties loaded on millions of pages per day, therefore once we know the domains that these hackers are using to send their stolen data, we can analyse the extent and impact of these operations. In this post we provide a post analysis of the four big breaches this year, plus some insights our data gives in on-going attacks.
```
Comment (remusao) [inline blog/trackers-who-steal.md]: > In this post we provide a post analysis

Shall we try to not repeat `post` in this sentence?

File: blog/trackers-who-steal.md [hunk 4]
```diff
@@ -0,0 +1,216 @@
...
+
+## Hack post-mortems
+
+### British Airways
+
+In September 2018, British Airways announced that a security breach had led to a large theft of customer data. [RiskIQ's](https://www.riskiq.com/blog/labs/magecart-british-airways-breach/) write up of the breach explains how the attackers compromised a script on the payment page, such that it would sent credit card information to a domain owned by the hackers: `baways.com`.
```
Comment (remusao) [inline blog/trackers-who-steal.md]: `it would sent` => `it would send`?

File: blog/trackers-who-steal.md [hunk 5]
```diff
@@ -0,0 +1,1225 @@
...
+</table>
+<p class="img-caption">Table 1: Sites affected by webfotce.me attack.</p>
+
+Compared to Ticketmaster, the impact of the breach on these sites is much smaller. Correllations between the dates of infection indicate that these sites were probably infected via a shared third-party (i.e. Inbenta) which was compromised. This shows how hackers can quickly achieve much greater scale by going for third-party services whose scripts will be loaded on many different sites.
+
+### NewEgg
+
+Like British Airways, NewEgg were hit by a [targetted attack](https://www.riskiq.com/blog/labs/magecart-newegg/). In this case the collection server was specific for the target. The hackers registered `neweggstats.com` in order to have a legitimate looking domain so that they could avoid suspicion for as long as possible.
+
+Looking at our data, we see that pages on `secure.newegg.com` were sending requests to `neweggstats.com` for just over a month, between 15th August and 18th September, with 90 pages affected in our dataset.
```
Comment (remusao) [inline blog/trackers-who-steal.md]: When you say "pages were affected", this is the number of distinct page loads which had this specific third-party loaded right? Or 90 different URLs?

File: blog/trackers-who-steal.md [hunk 6]
```diff
@@ -0,0 +1,1225 @@
...
+
+### VisionDirect
+
+On the 19th of November, VisionDirect, a large UK-based glasses retailer, [announced](https://www.visiondirect.co.uk/customer-data-theft) that their sites had been compromised between the 3rd and 8th of November. In this case, a script was injected into the page from `g-analytics.com` which pretending to be a Google Analytics script. This difference is, that it will also send credit-card numbers back when it sees them in the page.
```
Comment (remusao) [inline blog/trackers-who-steal.md]: "This difference is" => "The difference is"?

File: blog/trackers-who-steal.md [hunk 7]
```diff
@@ -0,0 +1,1225 @@
...
+
+Our analysis shows that VisionDirect's European sites (.fr, .it, .es, .co.uk, .eu and .nl) were all affected from the 3rd of November. On the .nl and .ie sites we still observed pages contacting the attacker's server on the 9th of November, suggesting that the malicious code may not have been completly removed as early as the press release suggests.
```
Comment (remusao) [inline blog/trackers-who-steal.md]: Should "completly" be spelled "completely"? Or is it a British spelling?

**Comment by sammacbeth** [comment]
Thanks @remusao for the comments - I've updated and fixed the typos.

**Pull request merged**